README.md

```diff
@@ -1 +1,30 @@
-"# todo_list" 
+# Awesome Books
```

Comment:
The project we work through this week is a simple to-do list app, and not the Awesome Books app. :bookmark: 

- [ ] Please update the name of your app to match the app you work on. :+1: